book/html.md

```diff
@@ -514,16 +514,16 @@ def add_tag(self, tag):
 Remember to use `tag` and `attribute` instead of `text` in `add_tag`,
 and try your parser again:
 
-``` {.example}
-<html>
-   <head>
-     <meta>
-     <link>
-     <link>
-     <link>
-     <link>
-     <link>
-     <meta>
+``` {.output}
+ <html>
```

Comment:
why new spaces?

Reply:
Our pretty-printer always prints a space between the indentation and the data.